public/modules/custom/tyollisyyspalvelut_common/tyollisyyspalvelut_common.module

```diff
@@ -14,13 +14,14 @@ use Drupal\Core\Entity\EntityForm;
  * @param Editor $editor
  */
 function tyollisyyspalvelut_common_ckeditor_css_alter(array &$css, Editor $editor) {
-  $css[] = drupal_get_path('module', 'tyollisyyspalvelut_common') . '/ckeditor.css';
+  $css[] = drupal_get_path('module', 'tyollisyyspalvelut_common') . 'css/ckeditor.css';
 }
```

Comment:
ckeditor.css file path is missing the first '/'. Also remember the remove ckeditor.css file from the old location. Now there's two ckeditor.css files.